### PR TITLE
Update docs README with instructions on locally previewing docs

### PR DESCRIPTION
**Pull request by snehankekre**
# What does this PR do?
This small PR updates the README in `/docs/` with instructions on how to use `doc-builder` to locally preview the documentation before submitting a PR. The current docs say previewing is not possible. However, the `doc-builder` [repo](https://github.com/huggingface/doc-builder#previewing) contains previewing instructions.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

Documentation: @sgugger @stevhliu

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -50,11 +50,26 @@ You can adapt the `--build_dir` to set any temporary folder that you prefer. Thi
 the MDX files that will be rendered as the documentation on the main website. You can inspect them in your favorite
 Markdown editor.
 
+## Previewing the documentation
+
+To preview the docs, use the following command:
```
Comment (sgugger) [inline docs/README.md:55]: I think you need extra packages for this, so maybe also indicate one has to do
```
pip install watchdog
```
before.

**Review by sgugger — APPROVED** [review]
Very nice, thanks a lot for updating those!

File: docs/README.md (hunk 1 above)
Comment (snehankekre) [reply docs/README.md:55]: Ah, I didn't realize `watchdog` needed to be installed separately. I think I either had it locally already or it was installed  by the command from earlier in the README:

```
pip install git+https://github.com/huggingface/doc-builder
```

Perhaps I already had it installed and that's why I didn't see the `ImportError`.

Comment (snehankekre) [reply docs/README.md]: Fixed on [cc231de](https://github.com/snehankekre/transformers/commit/cc231de39b0784bbea402699f0d040feb4980759).

**Comment by sgugger** [comment]
Thanks again for your contribution!

**Pull request merged**